module-4/backend/week_two.md

```diff
@@ -0,0 +1,43 @@
+## Week Two - Module 4 Recap
+
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+
+Note: When you're done, submit a PR. 
+
+1. What is Webpack and why is it useful?
+  - Webpack is a build tool that helps make a developer's life easier by bundling dependencies, compressing JS into one file, and making it easier to work with the browser.
+  
+2. When do you want to use event delegation?
+  - When there's a chance of events bubbling up
```

Comment:
also when you want events to be bound to elements that are not on the page on page load